pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -779,8 +780,9 @@ func makeFirewallName(name string) string {
 	return fmt.Sprintf("k8s-fw-%s", name)
 }
 
-func makeFirewallDescription(ipAddress string) string {
-	return fmt.Sprintf("KubernetesAutoGenerated_OnlyAllowTrafficForDestinationIP_%s", ipAddress)
+func makeFirewallDescription(serviceName, ipAddress string) string {
+	return fmt.Sprintf(`{"kubernetes.io/service-ip":"%s", "kubernetes.io/service-name":"%s"}`,
```

Comment:
The only thing to consider is that description might be used by user for other purposes too.
If we want to play nice, we should assume that this string could potentially be a substring of the larger description text.
In that case, should we have pre and post delimiters around the JSON to clearly indicate that kubernetes will parse anything within that ?